JuliaInterface/read.g

```diff
@@ -6,17 +6,17 @@
 ReadPackage( "JuliaInterface", "gap/JuliaInterface.gi");
 ReadPackage( "JuliaInterface", "gap/BindCFunction.gi" );
 
-dirs:= DirectoriesPackageLibrary( "JuliaInterface", "julia" );
+dirs_julia := DirectoriesPackageLibrary( "JuliaInterface", "julia" );
 
-JuliaIncludeFile( Filename( dirs, "gaptypes.jl" ) );
+JuliaIncludeFile( Filename( dirs_julia, "gaptypes.jl" ) );
 
-dirs:= DirectoriesPackageLibrary( "JuliaInterface", "../LibGAP.jl/src" );
+dirs_libgap := DirectoriesPackageLibrary( "JuliaInterface", "../LibGAP.jl/src" );
 
 if not IsBound( JULIAINTERNAL_LOADED_FROM_JULIA ) then
-    JuliaEvalString( Concatenation( "Base.include( Main,\"", Filename( dirs, "initialization.jl" ), "\")" ) );
+    JuliaEvalString( Concatenation( "Base.include( Main,\"", Filename( dirs_libgap, "initialization.jl" ), "\")" ) );
```

Comment:
Question (kinda orthogonal to this PR, i.e., not a blocker to this PR, but looking at this code again made me wonder):  Why do we includle `gaptypes.jl` before `initialization.jl`? Bedeutet das nicht, dass deren relative Reihenfolge unterschiedlich ist, je nach dem, ob man GAP von Julia aus lädt, oder umgekehrt?

Falls es dafür einen technischen Grund gibt, würde ich den gerne hier in einem Code-Kommentar dokumentieren. Falls es keinen Grund gibt, würde ich die Reihenfolge tauschen; denn auch wenn es gerade keinen Unterschied macht, so könnte es doch eines Tags zu seltsamen Differenzen im Verhalten führen, welche dann nervig zu debuggen sind.

Thoughts?

Reply:
No, there is no real necessity to include `gaptypes.jl` before `initialization.jl`. It is just historical in the order the files were written in. The files are orthogonal to each other. However, `gaptypes.jl` needs to be loaded before any other of the files.

Reply:
OK, then let's swap it (doesn't have to be in this PR, of course)